index.html

```diff
@@ -3,99 +3,108 @@
 layout: master
 ---
 
-<div class="flex flex-col items-center space-y-32 text-white pt-16">
-  <div class="flex flex-col items-center space-y-8">
-    <img src="assets/images/t-rex.png" />
-    <p class="font-bold text-4xl center">
+<div
+  class="flex flex-col items-stretch text-white pt-4 md:pt-6 lg:pt-4 space-y-16 md:space-y-24 lg:space-y-32"
+>
+  <div
+    class="hero flex flex-col items-center space-y-8 py-16 md:py-18 lg:py-28"
+  >
+    <div class="flame-demo h-28 md:h-40"></div>
+
+    <p
+      class="font-bold center font-accent text-center text-3xl md:text-4xl lg:text-6xl"
+    >
       Versatile, Open Source & Runs Everywhere
```

Comment:
Wasn't it Flexible that we agreed on? Or maybe we never settled that?

Reply:
Idk man, I am not changing content on this pr

Reply:
Ship it, and we'll do another content overhaul PR! (If there are more content things that should be changed)